.github/workflows/test-package.yml

```diff
@@ -0,0 +1,85 @@
+name: Dart CI
+
+on:
+  # Run on PRs and pushes to the default branch.
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 0 * * 0"
+
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  # Check code formatting and static analysis on a single OS (linux)
+  # against Dart dev.
+  analyze:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
A matrix w/ one entry is a bit...uh...too much, right?

Reply:
My assumption was that we'll add more versions when 2.12 stable is released and that the current "dev only" situation is a temporary oddity.

Reply:
Discussed offline, decided to keep it.